src/style.css

```diff
@@ -0,0 +1,32 @@
+.frame {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
you need to reset margin for last element in the row

Reply:
@anastasiia-tilikina  you mean:
 .stars :last-child {
  margin-right: 0;
}